sys/linux/io_uring.txt

```diff
@@ -489,3 +496,50 @@ io_uring_sqe_tee_misc {
 	splice_fd_in		fd
 	pad_unused		array[const[0, int64], 2]
 }
+
+io_uring_sync_cancel_reg {
+	addr	int64
+	fd	fd
+	flags	flags[iouring_async_cancel_flags, int32]
+	timeout	kernel_timespec
+	opcode	int8
+	pad	array[const[0, int8], 7]
+	pad2	array[const[0, int64], 3]
+}
+
+kernel_timespec {
+	tv_sec	int64
+	tv_nsec	int64
+}
+
+iouring_async_cancel_flags = IORING_ASYNC_CANCEL_ALL, IORING_ASYNC_CANCEL_FD, IORING_ASYNC_CANCEL_ANY, IORING_ASYNC_CANCEL_FD_FIXED, IORING_ASYNC_CANCEL_USERDATA, IORING_ASYNC_CANCEL_OP
+
+io_uring_file_index_range {
+	off	int32
+	len	int32
+	resv	int64
+}
+
+io_uring_buf_status {
+	buf_group	int32
```

Comment:
I think it should be of `io_uring_bgid[int32]` type and have `(in)` [direction](https://github.com/google/syzkaller/blob/master/docs/syscall_descriptions_syntax.md#structs).

Reply:
As I understand it, this is used as an index.